release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1739,6 +1743,34 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-management-console-bug-fixes"]
 ==== Management Console
 
+* Previously, alerts loaded from non-prometheus datasources such as logs. This caused the source of all alerts to be displayed always as *Prometheus*. With this update, alert sources are displayed correctly. (link:https://issues.redhat.com/browse/OCPBUGS-9907[*OCPBUGS-9907*])
```

Comment:
```suggestion
* Previously, alerts loaded from non-Prometheus datasources such as logs. This caused the source of all alerts to be displayed always as *Prometheus*. With this update, alert sources are displayed correctly. (link:https://issues.redhat.com/browse/OCPBUGS-9907[*OCPBUGS-9907*])
```
Looks like a brand name that needs a cap. https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1375785039